libcloud/httplib_ssl.py

```diff
@@ -187,17 +186,19 @@ def request(self, method, url, body=None, headers=None, raw=False):
             verify=self.ca_cert if self.ca_cert is not None else self.verify
         )
 
-    def prepared_request(self, method, url, body=None, headers=None, raw=False):
-        req = requests.Request(method, ''.join([self.host, url]), data=body, headers=headers)
-        
+    def prepared_request(self, method, url, body=None,
+                         headers=None, raw=False):
+        req = requests.Request(method, ''.join([self.host, url]),
+                               data=body, headers=headers)
+
         prepped = self.session.prepare_request(req)
 
         prepped.body = body
-        
-        self.response = self.session.send(prepped,
+
+        self.response = self.session.send(
+            prepped,
             stream=raw,
-            verify=self.ca_cert if self.ca_cert is not None else self.verify
-        )
+            verify=self.ca_cert if self.ca_cert is not None else self.verify)
```

Comment:
It can be a string or a boolean. This statement says to allow the cert or the flag http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification the string option is hidden in the docs. I did test it out